packages/readme.md

```diff
@@ -0,0 +1,233 @@
+# Structure of package.mk files
+
+## Introduction
+
+The package.mk file defines variables and functions to build a package.
+
+## Variables
+All avialible variable, to control the build behavoir of your package.
+Please use these in the order, listed here.
```

Comment:
To control the build behaviour of your package, use variables in the top-down order listed here.